tests/expected/panic/panic-2018/expected

```diff
@@ -1,10 +1,10 @@
-Failed Checks: concat!("Panic: {} code: ", 10), msg
+Failed Checks: concat! ("Panic: {} code: ", 10), msg
 Failed Checks: msg
 Failed Checks: explicit panic
 Failed Checks: Panic message
 Failed Checks: "Panic message with arg {}", "str"
 Failed Checks: "{}", msg
 
-Failed Checks: concat!("ArrayVec::", "try_insert",\
-": index {} is out of bounds in vector of length {}"),\
-5, 3
+Failed Checks: concat!\
```

Comment:
Is this new line on purpose?

Reply:
TBH, I have no idea how this new line came about. It's not clear how where those new lines are coming from.